app/characterization_services/mediainfo_characterization_service.rb

```
@@ -69,61 +75,6 @@ def valid?
 
   private
 
-    # Null Object modeling metadata values from MediaInfo.from
-    # @see MediaInfo::Tracks
-    class NullTracks
```

Is there a reason to keep NullTracks? It just silently passes the file on to derivative generation after MediaInfo fails.

---

If tests don't fail let's keep it out.